src/main/java/controllers/NotificationController.java

```diff
@@ -31,9 +32,6 @@ public String handleRequest(HttpServletRequest request, HttpServletResponse resp
         final List<Notification> notificationList = new ArrayList<>();
         notificationList.add(processor.processRequest(request));
         configuration.getTriggerEndpointConfigurations()
-                .stream()
-                .filter(endpointConf -> endpointConf.getTriggerEndpoint().equals("/app")) //we can pass stuff in
-                // request and compare to this here
                 .forEach(endpointConf -> endpointConf.getSendToEndpoints()
```

Comment:
Am I wrong or now the notifications are sent to all send-to endpoints registered (for multiple trigger-endpoints if present), no matter what was the trigger-endpoint? I think you still need to keep the filter in order to find the send-to endpoints appropriate for the trigger-endpoint to which the request was sent (the information in `request::getRequestURI`).

Reply:
that's true. It won't be that easy though, to filter using just `request::getRequestURI` since its value is `send-update` and that's not provided in configuration and blah blah. I will get into this on Wednesday